src/controllers/commentController.js

```diff
@@ -0,0 +1,96 @@
+import commentService from '../services/commentService.js';
+import mongoose from 'mongoose';
+
+export const createComment = async (req, res) => {
+  try {
+    console.log('Server:: create the Comment');
+    const id = req.params.threadId;
+    if (!mongoose.Types.ObjectId.isValid(id)) {
```

Comment:
I don't think this is necessary tho, you can try on the postman to have an invalid objectId.

Reply:
Without this, if we give an invalid threadId, postman reply "Cannot create this comment"; If have this, we will more clear about the error type